terraform/root_integration.tf

```diff
@@ -0,0 +1,102 @@
+  command         = "docker exec $(docker ps -aq -f ancestor=${module.ecr_jenkins_repository.repository.repository_url} -f status=running) /opt/backup.sh ${data.aws_ssm_parameter.jenkins_backup_healthcheck_url.value}"
+  ec2_instance_id = module.jenkins_ec2.instance_id
+  name            = "tdr-jenkins-backup-window"
+  schedule        = "cron(0 0 18 ? * MON-FRI *)"
+  common_tags     = local.common_tags
+}
+
+module "jenkins_maintenance_window_event" {
+  source                  = "./tdr-terraform-modules/cloudwatch_events"
+  event_pattern           = "jenkins_maintenance_event_window"
+  lambda_event_target_arn = list(data.aws_lambda_function.notifications_function.arn)
+  rule_name               = "jenkins-backup-maintenance-window"
+  rule_description        = "Capture failed runs of the jenkins backup"
+  event_variables         = { window_id = module.jenkins_backup_maintenance_window.window_id }
+}
+
+module "jenkins_integration_fargate_policy" {
+  source        = "./tdr-terraform-modules/iam_policy"
+  name          = "TDRJenkinsFargatePolicy${title(local.environment)}"
+  policy_string = templatefile("./tdr-terraform-modules/iam_policy/templates/jenkins_fargate_integration.json.tpl", { account_id = data.aws_caller_identity.current.account_id })
+}
+
+module "jenkins_integration_fargate_role" {
+  source             = "./tdr-terraform-modules/iam_role"
+  common_tags        = local.common_tags
+  assume_role_policy = templatefile("./tdr-terraform-modules/iam_policy/templates/assume_role_policy.json.tpl", { role_arn = module.jenkins_integration_ecs_task_role.role.arn })
+  name               = "TDRJenkinsFargateRole${title(local.environment)}"
+  policy_attachments = { fargate_policy = module.jenkins_integration_fargate_policy.policy_arn, ssm_core = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore" }
+}
+
+module "jenkins_certificate" {
+  source      = "./tdr-terraform-modules/certificatemanager"
+  project     = var.project
+  function    = "jenkins"
+  dns_zone    = var.dns_zone
+  domain_name = var.domain_name
+  common_tags = local.common_tags
+}
+
+module "jenkins_alb" {
+  source                           = "./tdr-terraform-modules/alb"
+  project                          = var.project
+  function                         = var.function
+  environment                      = local.environment
+  alb_log_bucket                   = module.jenkins_logs_s3.s3_bucket_id
+  alb_security_group_id            = module.jenkins_alb_security_group.security_group_id
+  certificate_arn                  = module.jenkins_certificate.certificate_arn
+  health_check_unhealthy_threshold = 5
+  domain_name                      = var.domain_name
+  public_subnets                   = module.jenkins_vpc.public_subnets
+  target_id                        = module.jenkins_ec2.instance_id
+  vpc_id                           = module.jenkins_vpc.vpc_id
+  common_tags                      = local.common_tags
+}
```

Comment:
Missing line break

Reply:
Still missing ... ;)

Reply:
I've added one in at the bottom which I think is the one you're talking about. Github keeps showing me old versions of things.